tests/fixtures/class-syntax.js

```diff
@@ -0,0 +1,7 @@
+class AppRouter extends Router {
+  @tracked location = config.locationType;
```

Comment:
Seems good to confirm decorators don’t blow up parsing (which I assume is the point), but just wanted to confirm that we are on the same page: it’s super unlikely that we’d allow this specific field as tracked, right?

Reply:
Correct, making this tracked would have no effect. I'd be happy to change the test if you can recommend an example that would be less confusing.

Reply:
Maybe just `@someDecorator` makes it clear that we're testing for abstract decorator support, not a specific expected feature?

Reply:
Ya, probably but its fine either way to me.